OpenRA.Utility/Program.cs

```diff
@@ -43,9 +43,16 @@ static void Main(string[] args)
 			Log.AddChannel("perf", null);
 			Log.AddChannel("debug", null);
 
-			Game.InitializeSettings(Arguments.Empty);
+			var arguments = new Arguments(args);
+			args = arguments.GetSkippedValues();
```

Comment:
If we go the `FirstOrDefault` route then the entire Arguments regex/dict is unnecessary.